schema/mysql/upgrades/1.0.0.sql

```diff
@@ -164,10 +164,14 @@ ALTER TABLE notification_history
   MODIFY `text` longtext NOT NULL;
 
 ALTER TABLE host_state
-  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state;
+  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state,
+  CHANGE attempt check_attempt tinyint unsigned NOT NULL,
+  CHANGE timeout check_timeout int unsigned DEFAULT NULL;
 
 ALTER TABLE service_state
-  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state;
+  ADD COLUMN previous_soft_state tinyint unsigned NOT NULL AFTER hard_state,
+  CHANGE attempt check_attempt tinyint unsigned NOT NULL,
+  CHANGE timeout check_timeout int unsigned DEFAULT NULL;
```

Comment:
I guess RENAME isn’t old enough?

Reply:
Yes, it' s actually only available since mysql 8.